Source/MoreDotNet.Test/Extensions/Common/RandomExtensions/NextCharTests.cs

```diff
@@ -0,0 +1,140 @@
+﻿namespace MoreDotNet.Tests.Extensions.Common.RandomExtensions
+{
+    using System;
+
+    using Moq;
+    using MoreDotNet.Extensions.Common;
+
+    using Xunit;
+
+    public class NextCharTests
+    {
+        [Fact]
+        public void NextChar_ShouldReturnChar()
```

Comment:
Missing test cases:
What happens if the random is null?
What happens if the NextChar(this Random random, CharType mode) is called with an invalid CharType?